krun/vm_defs.py

```diff
@@ -261,9 +261,9 @@ def _run_exec_popen(self, args, stderr_file=subprocess.PIPE):
         # command with. Command line arguments will have been appended *inside*
         # to adjust the new user's environment once the user switch has
         # occurred.
-        child_pipe = subprocess.Popen(args, stdout=subprocess.PIPE,
+        child_pipe = subprocess32.Popen(args, stdout=subprocess32.PIPE,
                                       stderr=stderr_file, env={})
-        return read_popen_output_carefully(child_pipe, platform=self.platform)
+        return read_popen_output_carefully(child_pipe, platform=self.platform, timeout=30)  # XXX
```

Comment:
@ltratt  See here, we only specify a timeout when running a process execution. This value will need to eventually come from a config file.